server-phoenix/priv/repo/migrations/20220525214039_change_column_names.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Helios.Repo.Migrations.ChangeColumnNames do
+  use Ecto.Migration
+
+  # If the current schema has column names created_at, then we will rename all of the created_at columns to inserted_at
+  def change do
+    execute("DO $$
+    BEGIN
+      IF EXISTS(SELECT *
+        FROM information_schema.columns
+        WHERE table_name='active_admin_comments' and column_name='created_at')
+      THEN
+        ALTER TABLE public.active_admin_comments RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE admin_users RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE admin_users RENAME COLUMN remember_created_at TO remember_inserted_at;
+        ALTER TABLE announcements RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE daily_event_summaries RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE developer_users RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE events RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE locations RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE traffic_cams RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE users RENAME COLUMN created_at TO inserted_at;
+        ALTER TABLE widgets RENAME COLUMN created_at TO inserted_at;
+      END IF;
+    END $$;")
+  end
+end
```

Comment:
We can skip `active_admin_comments`. 

`rename/3` would leverage the ecto migration DSL and be reversible:
https://hexdocs.pm/ecto_sql/Ecto.Migration.html#rename/3

Reply:
Resolved!